client/src/components/SignupForm/SignupForm.js

```diff
@@ -57,21 +89,34 @@ const ValidationPopup = ({ error, active }) => {
   )
 }
 
-function SignupForm (props) {
-  const { handleSubmit, signup } = props
+const SignupForm = (props) => {
```

Comment:
Noticed that you are preferring the `const X = ...` approach versus the `function X () {}`. As a team, we've been using the latter style. Let me know what you think!